src/plugins/spaces_oss/public/plugin.test.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { spacesApiMock } from './api.mock';
+import { SpacesOssPlugin } from './plugin';
+
+describe('SpacesOssPlugin', () => {
+  let plugin: SpacesOssPlugin;
+
+  beforeEach(() => {
+    plugin = new SpacesOssPlugin();
+  });
+
+  describe('#start', () => {
```

Comment:
Can we add tests for `#setup`, to verify that the API can be registered at most once?

Reply:
Good idea, I shouldn't have missed that! Done along with the other requested changes in ee41e6ba.